helm/charts/hydra/values.yaml

```diff
@@ -79,6 +79,8 @@ ingress:
 #        - api.hydra.local
 #      - secretName: hydra-api-example-tls
 
+# Use a pre-existing secret
+# existingSecret: my-preexisting-secret
```

Comment:
I think it makes sense to move this to `hydra` section (like `autoMigrate`). What do you think? Also, to make it maybe a bit clearer what this does, what do you think about the name `secretKeyRef`?

Reply:
Yes, I wasn't sure about the proper location. Will move it as per your suggestion.

Not sure about `secretKeyRef` though.
I would expect a `secretKeyRef` to refer to a secret _key_ (such as _secretsCookie_ or _secretsSystem_, not a secret _name_)

I went with existingSecret because it is used in a [fair amount of helm charts in the stable repository](https://github.com/helm/charts/search?q=secret+existing&unscoped_q=secret+existing) which makes it a convention of sorts.

Reply:
Oh, you're absolutely right! This looks good to me :)